guide/C/ch_reports.xml

```diff
@@ -88,6 +88,28 @@ Configuration.</para>
 price information in your file, the report will indicate an error. This report 
 includes extended information about commodity holdings, including information 
 about the basis, gain, and return of individual commodities.</para>
+
+  <sect4 id="rpt-advport-capgains">
+    <title>Advanced Portfolio Capital Gains</title>
+    <para>
+      The Advanced Portfolio report doesn’t use the capital gain splits to
+      calculate capital gains. It calculates the gains from the various buy and
+      sell transactions in the account without regard to whether the gains and
+      losses are recorded or not. Any realized gain splits are ignored.
```

Comment:
Why would changing the split make any difference? The APR looks afresh at everything every time it's run.